consensus/src/consensus/mod.rs

```diff
@@ -679,6 +681,11 @@ impl ConsensusApi for Consensus {
         }
     }
 
+    fn get_sink(&self) -> Option<Hash> {
+        // TODO: unwrap on virtual state read when staging consensus is implemented
```

Comment:
I think we can just unwrap now. Right @michaelsutton ?

Reply:
yes we can unwrap